src/index.ts

```diff
@@ -25,11 +25,11 @@ const helpers = require('./common/helpers');
 const swaggerUi = require('swagger-ui-express');
 
 const PORT = process.env.PORT || 5000;
-const DEBUG = process.argv.filter(val => val == 'dev').length > 0;
 const MOCK = process.argv.filter(val => val == 'mock').length > 0;
 
 app.use(bodyParser.urlencoded({ extended: true }));
 
+app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
Do we need this twice lol

Reply:
good catch again, probably some merging effect :)